Drop upstream/linux versions from xcb .so for dlopen(3) to find them

--- Telegram/lib_base/base/platform/linux/base_linux_xcb_library.cpp.orig	Thu Aug  6 22:59:39 2026
+++ Telegram/lib_base/base/platform/linux/base_linux_xcb_library.cpp	Thu Aug  6 23:00:03 2026
@@ -11,7 +11,7 @@
 namespace base::Platform::XCB::Library {
 
 void *LoadSymbol(const char *name) {
-	static const auto Handle = LoadLibrary("libxcb.so.1", RTLD_NODELETE);
+	static const auto Handle = LoadLibrary("libxcb.so", RTLD_NODELETE);
 	return Handle ? LoadSymbolGeneric(Handle, name) : nullptr;
 }
 
